src/android/com/megster/cordova/BluetoothSerial.java

```diff
@@ -565,9 +628,15 @@ private void connect(CordovaArgs args, boolean secure, CallbackContext callbackC
 
             }
 
+            if (bluetoothSerialService!=null) {
+                Log.d(TAG, "## Service STATE " + bluetoothSerialService.getState());
+            }
+
             if (bluetoothSerialService!=null && (bluetoothSerialService.getState() == BluetoothSerialService.STATE_CONNECTING
-                || bluetoothSerialService.getState() == BluetoothSerialService.STATE_CONNECTED)) {
+                || bluetoothSerialService.getState() == BluetoothSerialService.STATE_CONNECTED
+                || bluetoothSerialService.getState() == BluetoothSerialService.STATE_LISTEN)) {
```

Comment:
Added state LISTEN as the Accept thread is in the state listen until gets connected